### ISSUE TITLE
Gitea version outdated, 1.5.2/1.6.0

### ISSUE THREAD

**Issue by erenfro** [turnkeylinux/tracker#1231]
Gitea 1.5.0 is already getting outdated by newer releases of Gitea, now currently 1.6

**Comment by JedMeister** [comment]
Hi @erenfro - Thanks for the heads up. I'm not 100% sure when we'll rebuild it to include the latest version but I've made a note, so next lot of builds we do, I'll be sure to include Gitea.

In the meantime, you can update the version of Gitea included via the instruction [here](https://docs.gitea.io/en-us/install-from-binary/). Scroll down about halfway and you'll see a heading "Updating to a new version".

Before doing an update, ensure that you have a backup (via TKLBAM and/or via the [Gitea instructions](https://docs.gitea.io/en-us/backup-and-restore/)).

Once you have your backup (ideally tested to make sure it's complete), essentially the update should be a simple case of stopping Gitea:
```
service gitea stop
```
Then downloading the latest binary and replacing the current one (I recommend renaming the existing one just in case you want to roll back):
```
VERSION="1.6.0" # update as appropriate
mv /usr/local/bin/gitea /usr/local/bin/gitea.bak
wget -O /usr/local/bin/gitea https://dl.gitea.io/gitea/$VERSION/gitea-VERSION-linux-amd64
```

Note that for security, you may also wish to confirm that the binary is signed. Instructions are [here](https://docs.gitea.io/en-us/install-from-binary/) - Under the heading "Verify GPG signature".

**Comment by JedMeister** [comment]
FWIW when you initially posted, v1.6.0 was still in development. We rebuilt the appliance about 3 weeks ago including the latest stable release at that time - v1.5.3 (v1.6.0 was still RC). I notice that v1.6.0 was released a few days after our rebuild and they're just released v1.6.1 last week.

I'm not sure when it will be rebuilt again, but hopefully not too far away as I do see a few improvements (including security) that would be good to have included. However I'm going to close this issue for now though as there I can't see any glaring security issues/vulnerabilities associated with the v1.5.3 release at this point.
